ansible_runner/display_callback/module.py

```diff
@@ -21,18 +21,35 @@
 import collections
 import contextlib
 import datetime
+import os
 import sys
 import uuid
 from copy import copy
 
 # Ansible
 from ansible import constants as C
 from ansible.plugins.callback import CallbackBase
-from ansible.plugins.callback.default import CallbackModule as DefaultCallbackModule
+from ansible.plugins.loader import callback_loader
 
-# AWX Display Callback
 from .events import event_context
-from .minimal import CallbackModule as MinimalCallbackModule
+
+user_specified_callback = os.getenv('ORIGINAL_STDOUT_CALLBACK')
+runner_callback = os.environ.pop('ANSIBLE_STDOUT_CALLBACK', 'awx_display')
```

Comment:
I don't think there's any case where this would be something other than `'awx_display'`? If that's true, then I think it would be easier to follow to hard-code that.